recipe/meta.yaml

```diff
@@ -44,6 +47,9 @@ requirements:
     - nccl              {{ nccl        }}
     - tensorflow        {{ tensorflow  }}
     - pytorch           {{ pytorch }}
+    # Use pins to control cos6/cos7 match
+    - libgcc-ng  {{ libgcc }}
+    - libstdcxx-ng  {{ libstdcxx }}
   run:
```

Comment:
This one needs an update to have the pin in `run` as well

Reply:
Yes, adding it.